data/human/human missions.txt

```diff
@@ -7453,3 +7453,411 @@ mission "Adelita 5"
 			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
 			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
 			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`
+
+
+# Part 2
+# Best case: 10m, 4100 per day best case out of the negative after ~6.7 years
+# Worst case: 11m, 2000 per day worst case out of the negative after ~15 years (13.7 years, plus a little longer for 10M-->11M)
+# Super Worst case: 11m, 2000 per day worst case out of the negative after over 40 years if you don't pay off the debt
+# Driven by if you were mean to Howard in part 1 and part 2, which of 3 investment options picked, and if you pay off debt ASAP
+
+mission "Syndicate Business 3"
+	name "Visit Howard on <planet>"
+	description "Howard wants to tell you about another investment opportunity."
+	source
+		government "Syndicate"
+		attributes "core"
+		not planet "Hephaestus"
+	destination "Hephaestus"
+	to offer
+		has "Syndicate Business 2: done"
+		random < 40
+		credits > 40000000
+	on offer
+		conversation
+			`After an uneventful trip to the spaceport you return to the <ship> and see a message from Howard, the Megaparsec salesman who sold you a one million credit bond. The message says:`
+			`	"<first> <last>! My bank contact tells me you've been doing quite well for yourself, and I've come across an even bigger investment opportunity than last time! Meet me at my new office on <planet> (note the address, I've moved up since my promotion). Looking forward to chatting with you!"`
+			`		Howard Vendeur`
+			`		Investor Relations Manager, Megaparsec Incorporated`
+			choice
+				`	(Message him back that you're on the way.)`
+				`	(Send a message congratulating him on the promotion.)`
+				`	(Ignore and delete the message.)`
+					decline
+			`	You immediately receive an automated response:`
+			`	"Dear Valued Contact,`
+			`	Thank you for reaching out. I am currently out of the office and will respond to your message as soon as possible.`
+			`	For urgent matters, please contact our support team."`
+			`		Howard Vendeur`
+			`		Investor Relations Manager, Megaparsec Incorporated`
+				accept
+	on complete
+		conversation
+			`You land the <ship> on <planet> and make your way to Howard's office at the Megaparsec Incorporated satellite office near the Syndicate's headquarters.`
```

Comment:
I think this should still be `<planet>` because otherwise you'd land on Hephaestus and it would say that your ship landed on whatever planet the mission was offered at rather than Hephaestus . I also picked Hephaestus because it was the galactic home of Syndicated Systems, so what about something like:

```suggestion
			`You land the <ship> on <planet> and make your way to Howard's office at the Megaparsec Incorporated satellite office near the galactic headquarters of Syndicated Systems.`
```

Reply:
For some reason I thought this was in the `on offer` rather than `on complete`. Sorry.

Reply:
All good!